docs/configure-custom-values-scalardb-analytics-postgresql.md

```diff
@@ -0,0 +1,185 @@
+```
+
+### Affinity configurations (recommended in production environments)
+
+To control pod deployment by using affinity and anti-affinity in Kubernetes, you can use `scalardbAnalyticsPostgreSQL.affinity`.
+
+You can configure affinity and anti-affinity by using the same syntax for affinity and anti-affinity in Kubernetes. For more details on configuring affinity in Kubernetes, see [Assigning Pods to Nodes](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/).
+
+```yaml
+scalardbAnalyticsPostgreSQL:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/name
+                operator: In
+                values:
+                  - scalardb-analytics-postgresql
+              - key: app.kubernetes.io/app
+                operator: In
+                values:
+                  - scalardb-analytics-postgresql
+          topologyKey: kubernetes.io/hostname
+```
+
+### SecurityContext configurations (default value is recommended)
+
+To set SecurityContext and PodSecurityContext for ScalarDB Analytics with PostgreSQL pods, you can use `scalardbAnalyticsPostgreSQL.securityContext`, `scalardbAnalyticsPostgreSQL.podSecurityContext`, and `schemaImporter.securityContext`.
+
+You can configure SecurityContext and PodSecurityContext by using the same syntax as SecurityContext and PodSecurityContext in Kubernetes. For more details on the SecurityContext and PodSecurityContext configurations in Kubernetes, see [Configure a Security Context for a Pod or Container](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/).
+
+```yaml
+scalardbAnalyticsPostgreSQL:
+  podSecurityContext:
+    fsGroup: 201
+    seccompProfile:
+      type: RuntimeDefault
+  securityContext:
+    capabilities:
+      drop:
+        - ALL
+    runAsNonRoot: true
+    runAsUser: 999
+    allowPrivilegeEscalation: false
+
+schemaImporter:
+  securityContext:
+    capabilities:
+      drop:
+        - ALL
+    runAsNonRoot: true
+    allowPrivilegeEscalation: false
+```
+
+### Image configurations (default value is recommended)
+
+If you want to change the image repository, you can use `scalardbAnalyticsPostgreSQL.image.repository` and `schemaImporter.image.repository` to specify the container repository information of the ScalarDB Analytics with PostgreSQL and Schema Importer images that you want to pull.
+
+```yaml
+scalardbAnalyticsPostgreSQL:
+  image:
+    repository: <SCALARDB_ANALYTICS_WITH_POSTGRESQL_CONTAINER_IMAGE>
+
+schemaImporter:
+  image:
+    repository: <SCHEMA_IMPORTER_CONTAINER_IMAGE>
+```
+
+### Replica configurations (optional based on your environment)
+
+You can specify the number of ScalarDB Analytics with PostgreSQL replicas (pods) by using `scalardbAnalyticsPostgreSQL.replicaCount`.
+
+```yaml
+scalardbAnalyticsPostgreSQL:
+  replicaCount: 3
```

Comment:
Does this setting create the Postgres instances with replication? Or does it just make multiple instances?

Reply:
This setting just makes multiple instances without streaming replication or logical replication. At the moment, we cannot control the replication or HA feature of PostgreSQL by using this chart. We just deploy it as a single instance or multiple instances.

As I mentioned, we cannot control HA features, however, I think ScalarDB Analytics with PostgreSQL is basically a `read-only` product for the analytical workload. So, I don't think we need to use the replication feature at this time.

But, from the perspective of availability, I think we can make it more available by deploying 3 pods (ideally across on 3 zones in a cloud environment).

This is why I set `3` by default in this chart.